pkg/siftool/add_test.go

```diff
@@ -50,3 +60,49 @@ func Test_command_getAdd(t *testing.T) {
 		})
 	}
 }
+
+func Test_Add_Options(t *testing.T) {
```

Comment:
This seems like it's testing `getDigestFromInputFile` rather than `add` options?

```suggestion
func Test_getDigestFromInputFile(t *testing.T) {
```